core/arch/arm/plat-nuvoton/main.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2016-2023, Linaro Limited
+ *
+ * Copyright (c) 2014-2023, STMicroelectronics International N.V.
+ *
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arm.h>
+#include <console.h>
+#include <drivers/gic.h>
+#include <drivers/ns16550.h>
+#include <drivers/tzc400.h>
+#include <initcall.h>
+#include <keep.h>
+#include <kernel/boot.h>
+#include <kernel/interrupt.h>
+#include <kernel/misc.h>
+#include <kernel/panic.h>
+#include <kernel/tee_time.h>
+#include <mm/core_memprot.h>
+#include <mm/core_mmu.h>
+#include <npcm845x_utils.h>
+#include <platform_config.h>
+#include <sm/psci.h>
+#include <stdint.h>
+#include <string.h>
+#include <trace.h>
+
+static struct gic_data gic_data __nex_bss;
+static struct ns16550_data console_data __nex_bss;
+
+register_phys_mem_pgdir(MEM_AREA_IO_SEC, CONSOLE_UART_BASE, UART_REG_SIZE);
+
+#ifdef DRAM0_BASE
```

Comment:
check your _platform_config.h_: DRAM0_BASE is always defined and DRAM1_BASE is never defined. Please remove uncessarry conditions/code.